pkg/cli/cc/health.go

```diff
@@ -13,7 +13,7 @@ import (
 // ServiceHealthStatus adopted from grpc-health-probe cli implementation
 // https://github.com/grpc-ecosystem/grpc-health-probe/blob/master/main.go.
 func ServiceHealthStatus(service string) bool {
-	addr := "localhost:9494"
+	addr := "0.0.0.0:9494"
```

Comment:
Can we get this address from the topaz configuration file ?

Reply:
@carabasdaniel I would like to be able to do this
```
	cfg := config.CurrentConfig()
	addr, _ := cfg.HealthService()
```
which creates an import cycle as config/generator.go imports package cc

Reply:
Since ServiceHealthStatus is only called by template install
Probably better to make the change there like this:
```
// 4 - wait for health endpoint to be in serving state
	cfg := config.CurrentConfig()
	addr, _ := cfg.HealthService()
	if !cc.ServiceHealthStatus(addr, "") {
		return fmt.Errorf("gRPC endpoint not SERVING")
	}
```